bcal.c

```diff
@@ -784,6 +847,38 @@ int main(int argc, char **argv)
 		printhex_u128(offset);
 		fprintf(stdout, "\n");
 	}
+	
+	/*Addition Operation*/
+	if (argc - optind == 5 && !strcmp(argv[3], "+") ) { 		
```

Comment:
As of now 2 operands is fine. More like a PoC. But we have to have a full-blown parser for scanning inputs. That would be fun. I will share some sample code for the parsing.